hw/ip/otbn/doc/dv/index.md

```diff
@@ -173,6 +173,21 @@ See the instruction counter saturate.
 
 > This is tracked in the `insn_cnt_if` interface with the `InsnCntSaturated_C` cover property.
 
+#### External (bus-accessible) CSRs
+
+The OTBN block exposes functionality to a bus host through bus-accessible CSRs.
+Behavior of some CSRs depends on [OTBN's operational state]({{< relref ".#design-details-operational-states" >}}).
+
+Events we want to see:
+* For the `CMD` CSR, we want to see all valid commands being written crossed with all operational states to ensure commands are ignored as expected when OTBN is busy or locked.
+* For the `STATUS` CSR we want to see a read of all valid status codes.
+* For the `ERR_BITS` CSR, we want to see that every valid bit is read at least once, and we want to see an access in each operational state.
+* For the `FATAL_ALERT_CAUSE` CSR, we want to see that every valid bit is read at least once, and we want to see an access in each operational state.
+* For `INSN_CNT` we want to see a read returning a zero and a non-zero value.
+  We also want to see an access to this CSR in every operational state.
```

Comment:
Do you really mean "an access in each operational state" here? I would have expected both a read and a write?

Reply:
INSN_CNT (like all other registers in OTBN) are either read-only or write-only from software (at least currently, this might change depending on the outcome of #8287). So for INSN_CNT, I'm looking for a read (from software) in every state. (Or am I misunderstanding your question?)

Reply:
Either way, I replaced *access* with *read* now to use uniform naming and reduce the confusion (hopefully):

Reply:
That makes perfect sense. I was pre-empting things, thinking about the pending INSN_CNT / ERR_BITS changes.

Reply:
Sorry for the confusion. (I was already thinking about this upcoming change as well, that's why the access type is already passed to `on_ext_csr_access()`.)